.github/workflows/tests.yml

```diff
@@ -104,6 +104,21 @@ jobs:
       - run: make slim-builds
       - run: make test-shutdown
 
+  test-cli:
+    name: CLI - Linux
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - run: sudo bash scripts/environment/bootstrap-ubuntu-20.04.sh
+      - run: bash scripts/environment/prepare.sh
+      - run: make slim-builds
+      - run: make test-cli
```

Comment:
Do you think we could run this right after the `test` job above? It would save us a lot of CI time. :)

Reply:
Yea, we could bundle it with shutdown tests.